src/main/java/org/folio/dew/config/feign/CustomFeignErrorDecoder.java

```diff
@@ -14,6 +17,8 @@ public Exception decode(String methodKey, Response response) {
     String requestUrl = response.request().url();
     if (HttpStatus.NOT_FOUND.value() == response.status()) {
       return new NotFoundException(requestUrl);
+    } else if (HttpStatus.INTERNAL_SERVER_ERROR.value() == response.status()) {
+      return new BulkEditException(CANNOT_GET_RECORD_FROM_INVENTORY.formatted(requestUrl, response.reason()), ErrorType.ERROR);
```

Comment:
what about users?

Reply:
It works for users, I will rename the constant.